CodeEdit/Documents/WorkspaceDocument.swift

```diff
@@ -368,6 +380,89 @@ import TabBar
 
         super.close()
     }
+
+    /// Determines the windows should be closed.
+    ///
+    /// This method iterates all editied documents If there are any editied documents.
```

Comment:
Typo: "editied" should be "edited" #picky

Reply:
I fixed this typo.